pluma/dialogs/pluma-preferences-dialog.c

```diff
@@ -199,6 +206,88 @@ auto_indent_checkbutton_toggled (GtkToggleButton        *button,
 	pluma_prefs_manager_set_auto_indent (gtk_toggle_button_get_active (button));
 }
 
+static void
+draw_spaces_checkbutton_toggled (GtkToggleButton        *button,
+                                 PlumaPreferencesDialog *dlg)
+{
+	int v;
+        pluma_debug (DEBUG_PREFS);
+
+        g_return_if_fail (button == GTK_TOGGLE_BUTTON (dlg->priv->draw_spaces_checkbutton));
+
+        if (gtk_toggle_button_get_active (button))
+                v = 2;
+        else
+                v = 0;
```

Comment:
See comments above regarding `enum` on the gschema file. These values should also use an enum, otherwise it becomes very confusing to follow the code...
```c
typedef enum
{
  DRAW_NONE = 0,
  DRAW_TRAILING = 1,
  DRAW_ALL = 2
} DrawSpacesSettings;

draw_spaces_checkbutton_toggle (...)
{
  DrawSpacesSettings setting;
  if (...)
    setting = DRAW_ALL;
  else
    setting = DRAW_NONE;
  // ...
  gtk_widget_set_sensitive (..., setting > DRAW_NONE);
  // etc...
}
```
or something like that...

Reply:
Ok, I did